src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostUnclaimByIdControllerCFTTest.java

```diff
@@ -345,61 +362,112 @@ public void should_return_a_forbidden_when_unclaiming_a_task_by_id_with_differen
         common.cleanUpTask(taskId);
     }
 
-
     @Test
-    public void should_return_a_403_when_the_user_did_not_have_sufficient_permission_region_did_not_match() {
+    public void should_return_a_204_when_unclaiming_a_task_by_id_user_credentials_gp_flag_on() {
```

Comment:
can we update the test name so it's clear that its covering scenario where we are  "unclaiming for same user"?

e.g. should_return_a_204_when_unclaiming_a_task_by_id_with_same_user_credentials_gp_flag_on()